docker/package/tezos_setup_wizard.py

```diff
@@ -111,7 +110,7 @@ def is_full_snapshot(import_mode):
     prompt="Do you want to set up baking or to run the standalone node?",
     help="By default, tezos-baking provides predefined services for running baking instances "
     "on different networks.\nSometimes, however, you might want to only run the Tezos node.\n"
-    "When this option is chosen, this wizard will help you bootstrap tezos-node only.",
+    "When this option is chosen, this wizard will help you bootstrap octez-node only.",
```

Comment:
Meh, I don't know what's better, but I suggest using:
```suggestion
    "When this option is chosen, this wizard will help you bootstrap the Tezos node only.",
```
given that the Tezos node is all that's mentioned before (just a line above) and there's not need to make this more confusing for now.